src/bun.js/bindings/bindings.zig

```diff
@@ -421,6 +421,16 @@ pub const ZigString = extern struct {
             };
         }
 
+        /// Creates a `Slice` from bytes without an allocator, aka a string view.
+        /// This slice is only valid for the lifetime of the given input, and
+        /// will never be freed by this slice.
+        pub fn initStatic(input: []const u8) Slice {
```

Comment:
`fromUTF8NeverFree`?

Reply:
Otherwise, if this a latin1/utf16 string, it's invalid memory. ZigString.Slice is supposed to always be UTF8.

Reply:
thanks